src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -1000,6 +1000,12 @@ public void AnalyzeCommandBase_CachesNotificationsWhenPersistingToLogFile()
 
             testCase.Verbose = true;
             RunResultsCachingTestCase(testCase);
+
+            testCase.Files = ComprehensiveKindAndLevelsByFilePath;
+            RunResultsCachingTestCase(testCase, multithreaded: true);
+
+            testCase.Verbose = true;
```

Comment:
>testCase.Verbose = true; [](http://example.com/codeflow?start=12&length=24)

Wasn't verbose already set to true? Is this doing anything?

Reply:
fixed the logic.
I have to set the new files list + verbose to false. and, on line 1008, it will update verbose to true.